demo/package.json

```diff
@@ -11,7 +11,7 @@
     "storybook:axeServer": "axe-storybook --storybook-address http://localhost:6007"
   },
   "dependencies": {
-    "@chanzuckerberg/axe-storybook-testing": "*",
+    "@chanzuckerberg/axe-storybook-testing": "file:..",
```

Comment:
Fix not directly related to publishing. But I needed to fix it for publishing to succeed.

- This package.json is for a demo app, and is in an npm workspace for this repo
- Setting the version to `*` for another workspace should resolve to the local file system
- But... axe-storybook-testing itself is **_not_** in a workskpace
- So instead this was pulling any version from the npm registry

Pointing to the local file system so the `npm install` in the release job doesn't try to pull this from the github package registry (where it doesn't exist yet).